stdlib/inspect.pyi

```diff
@@ -154,16 +154,23 @@ if sys.version_info >= (3, 10):
         eval_str: bool = ...,
     ) -> Dict[str, Any]: ...
 
-# The name is the same as the enum's name in CPython
-class _ParameterKind(enum.IntEnum):
-    POSITIONAL_ONLY: int
-    POSITIONAL_OR_KEYWORD: int
-    VAR_POSITIONAL: int
-    KEYWORD_ONLY: int
-    VAR_KEYWORD: int
-
-    if sys.version_info >= (3, 8):
-        description: str
+if sys.version_info >= (3, 0):
```

Comment:
Nit: `enum` was introduced in Python 3.4, so it should either be `>= (3, 4)` or `>= (3,)` (since we only support 3.6+).

Reply:
Updated.